client/analysis/output.go

```diff
@@ -228,6 +247,14 @@ func printAllSummary(analysis types.Analysis) {
 		fmt.Printf("[HUSKYCI][SUMMARY] Low: %d\n", outputJSON.Summary.YarnAuditSummary.LowVuln)
 	}
 
+	if outputJSON.Summary.GitleaksSummary.FoundVuln || outputJSON.Summary.GitleaksSummary.FoundInfo {
+		fmt.Println()
+		fmt.Printf("[HUSKYCI][SUMMARY] Git -> %s\n", gitleaksVersion)
```

Comment:
@abzcoding could you change from `Git` to `Generic` as we could see it as the **type** of security test?

Reply:
thank you for spotting that, I will update it ASAP!